Widgets/Widgets/SearchWidget.swift

```diff
@@ -0,0 +1,134 @@
+        .contentMarginsDisabled()
+        .containerBackgroundRemovable(false)
+    }
+}
+
+// MARK: - Timeline Entry
+
+struct SearchEntry: TimelineEntry {
+    let date: Date
+    let configuration: SearchWidgetConfiguration
+    
+    init(date: Date = Date(), configuration: SearchWidgetConfiguration = SearchWidgetConfiguration()) {
+        self.date = date
+        self.configuration = configuration
+    }
+}
+
+// MARK: - Configuration
+
+struct SearchWidgetConfiguration {
+    let languageCode: String
+    let siteURL: URL
+    
+    init() {
+        let sharedCache = SharedContainerCache(fileName: SharedContainerCacheCommonNames.widgetCache)
+        let cache = sharedCache.loadCache() ?? WidgetCache(settings: .default, featuredContent: nil)
+        self.languageCode = cache.settings.languageCode
+        self.siteURL = cache.settings.siteURL
+    }
+}
+
+// MARK: - Timeline Provider
+
+struct SearchProvider: TimelineProvider {
+    typealias Entry = SearchEntry
+    
+    func placeholder(in context: Context) -> SearchEntry {
+        return SearchEntry()
+    }
+    
+    func getSnapshot(in context: Context, completion: @escaping (SearchEntry) -> Void) {
+        let entry = SearchEntry()
+        completion(entry)
+    }
+    
+    func getTimeline(in context: Context, completion: @escaping (Timeline<SearchEntry>) -> Void) {
+        let entry = SearchEntry()
+        let timeline = Timeline(entries: [entry], policy: .never)
+        completion(timeline)
+    }
+}
+
+// MARK: - View
+
+struct SearchWidgetView: View {
+    @Environment(\.widgetFamily) private var widgetFamily
+    @Environment(\.colorScheme) private var colorScheme
+    
+    var entry: SearchEntry
+    
+    private var theme: Theme {
+        return colorScheme == .dark ? Theme.widgetDark : Theme.widgetLight
+    }
+    
+    private var searchURL: URL? {
+        var components = URLComponents()
+        components.scheme = "wikipedia"
+        components.host = "search"
+        components.path = "/"
+        return components.url
+    }
+    
+    var body: some View {
+        VStack(spacing: 12) {
+            Image("wikipedia")
+                .resizable()
+                .aspectRatio(contentMode: .fit)
+                .frame(width: 70, height: 70)
+                .padding(.bottom, 8)
+            
+            HStack(spacing: 8) {
+                Image(systemName: "magnifyingglass")
+                    .font(.system(size: 14))
+                    .foregroundColor(Color(theme.colors.secondaryText))
+                
+                Spacer()
+            }
+            .padding(.horizontal, 12)
+            .padding(.vertical, 8)
+            .background(Color(theme.colors.searchFieldBackground))
+            .cornerRadius(8)
+            
+        }
+        .padding(16)
+        .frame(maxWidth: .infinity, maxHeight: .infinity)
+        .background(Color(theme.colors.paperBackground))
+        .widgetURL(searchURL)
```

Comment:
The url here should come from the widget entry, like `entry.url` 
It's not opening the Search tab on the app, it opens the app, but that's it
Try URL(string: "wikipedia://search")